### ISSUE TITLE
After installing 14.0.0, ncu -g stopped showing new versions

### ISSUE THREAD

**Issue by MartinManev** [raineorshine/npm-check-updates#1140]
```
$ ncu -g
[====================] 4/4 100%

 @vue/cli            5.0.4  →   5.0.5     
 corepack           0.10.0  →  0.11.2     
 npm-check-updates  13.1.2  →  14.0.0
 
ncu itself cannot upgrade global packages. Run the following to upgrade all global packages: 

npm -g install @vue/cli@5.0.5 corepack@0.11.2 npm-check-updates@14.0.0 npm@8.12.2
 
 $ npm -g install npm-check-updates
changed 273 packages, and audited 274 packages in 8s

29 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

$ ncu -g

No dependencies.

$ npm outdated -g
Package   Current  Wanted  Latest  Location               Depended by
@vue/cli    5.0.4   5.0.5   5.0.5  node_modules/@vue/cli  global
corepack   0.10.0  0.11.2  0.11.2  node_modules/corepack  global
```

Afterwards I installed `13.1.2`, `13.1.3`, `13.1.4` and `13.1.5` and `ncu -g` works correctly for each.

**Comment by raineorshine** [comment]
Thanks for reporting. I am investigating.

**Comment by raineorshine** [comment]
Fixed and published in `v14.0.1`
